README.md

```diff
@@ -1,23 +1,69 @@
-# Blog for *ASET ALiAS*
+# _ASET ALiAS Blog_
 
+This is the repository of the official blog for _Amity Linux Assitance Sapience_
 
-This static website has been created using hugo.
+ALiAS is a platform for upcoming developers for finding exposure by meeting the people working in related industries, learning various languages and becoming a better developer.
 
-*for hugo documentation [click here](https://gohugo.io/documentation/)*
+This website has been created using hugo and is currently under construction. Please contribute as much as you can. All your inputs are highly appreciated.
 
-*to get to know this theme better [click here](https://github.com/appernetic/hugo-nederburg-theme/)*
+* * *
 
-*markdown cheat sheet [click here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code)*
+## _Using the Website_
 
----
+### _For General users:_
 
-*to run this website on your system, clone the repository*
+`Hugo Installation`
 
-    >git clone https://github.com/rohanvtk/ALiAS_Blog.git
+1) [Click Here](https://github.com/gohugoio/hugo/releases) to  download latest version of Hugo for your operating system.
```

Comment:
The installation procedure needs to be pinpoint and bluntly accurate. Go like this. 

## Hugo Installation

**Windows**
Steps

**Linux**
More steps

**MacOS**
Suprise more steps.

Put the `code` and `commands` under preformatted like `this`
Use horizontal rules to divide sections.

Reply:
Will be updated soon.